src/terra-dev-site/contributing/GitWorkflow.b.contributing.md

```diff
@@ -0,0 +1,112 @@
+# Git Workflow
+The follow section outlines our Git workflow for Terra UI projects.
+
+## Fork The Repository
+
+Start by [forking](https://help.github.com/articles/fork-a-repo/) the repo you’ll be working on.
+
+_Note: The [terra team](https://github.com/orgs/cerner/teams/terra) and [terra contributors](https://github.com/orgs/cerner/teams/terra-contributors) can ignore this step and clone the repo directly._
+
+## Clone The Repo
+
+Example:
+
+`git clone git@github.com:cerner/terra-core.git`
+
+## Clone Your Forked Repo
+
+New contributors will need to clone their forked repo.
+
+Example:
+
+`git clone git@github.com:<github-username>/terra-core.git`
+
+## Creating A Branch
+
+Before starting your work, first ensure you're in the master branch and that you've pulled down the latest changes:
+
+```sh
+git checkout master
+git pull
+```
+
+Next, create a new branch for your work, with an appropriate name for your change:
+
+`git checkout -b my-feature-or-bug-branch`
+
+## Development Workflow
+
+Read through our [development workflow guide](#/contributing/terra-ui/developer-workflow) to better understand the development process.
+
+## Adding, Committing, And Pushing Changes
+
+Regularly git commit locally and git push to the remote branch. [Write good commit messages](http://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html).
+Your commits will automatically be squashed when your PR is merged into master.
+
+```sh
```

Comment:
Maybe add this before the example:
> This will commit your changes to your local copy of the repo.

Reply:
Updated: https://github.com/cerner/terra-ui/pull/108/commits/453b149ee491fa9a8d004c85dc550ca83dedf36d